db/schema.rb

```diff
@@ -1,58 +0,0 @@
-# This file is auto-generated from the current state of the database. Instead
```

Comment:
Why are you deleting this file?

Reply:
Deleting schema? In a real project, people do not do that!